### ISSUE TITLE
edit agario client

### ISSUE THREAD

**Issue by kurama129** [pulviscriptor/agario-client#172]
can you set client.auth_token to null and after connect do client.send with packet that sent here
https://github.com/pulviscriptor/agario-client/blob/master/agario-client.js#L108
And report if it helps, then i may create some function like client.auth(key).

**Comment by kurama129** [comment]
and send me on kagingforfun@gmail.com

**Comment by Wartoz** [comment]
need a fix fb bots don't connect :(

**Comment by pulviscriptor** [comment]
As i understand all reports about FB bots related to hacked accounts that are blocked by agar. On protocol level `agario-client` works fine.
So nothing i can do with it.
Correct me if i am wrong.

**Comment by Nick8fr32** [comment]
Its because all is here to ask if u Can make it so it only connect the auth to the client that is connected to a server instead of using the token before the bots even connects or that what i think they are all here for @pulviscriptor

**Comment by pulviscriptor** [comment]
@Nick8fr32 
Story starts here https://github.com/pulviscriptor/agario-client/issues/169
sejo153 says that someone says that sending auth after connect(?) will help anything and i need confirmation of this.
When i test my cookies they work every time so i can't reproduce problem to test/solve it.
I need reports from users who have FB problems. Somebody needs to try this https://github.com/pulviscriptor/agario-client/issues/169#issuecomment-214521824 and report if it works or not.

**Comment by sejo153** [comment]
http://prnt.sc/axxbba
correct me if i done wrong, after i edit to null it gives that error

**Comment by Flathe** [comment]
@Pulviscriptor
Hey, I try to use Facebook bots, but there's no facebook bot mass, i tried what u said on #169  but nothing change, only normal bots, (i just created thoses cookies, and make them lvl 5)
Thx.

**Comment by sejo153** [comment]
@pulviscriptor There you go what happens i edit this.auth_token to null in agario-cilent
And this happens i record video of cmd-- http://www.speedyshare.com/hnKEt/bandicam-2016-04-28-08-00-09-001.avi

**Comment by Sonicmaster13** [comment]
@Pulviscriptor I wish it was possible to ban the word 'Facebook' from issues... people keep spamming about modding your client to make their own facebook bots then not even crediting you in their release of it.

**Comment by kurama129** [comment]
hello sonic master im your fan  can i get your skyp and i know people are not crediting him

**Comment by pulviscriptor** [comment]
Duplicate of #169
